tests/testsuite.src/run_subscripts.at

```diff
@@ -642,7 +643,7 @@ AT_DATA([prog.cob], [
        END PROGRAM SUBN.
 ])
 
-AT_CHECK([$COMPILE -x -std=mf -debug -Wall -debug -O prog.cob ], [0], [], [prog.cob:25: warning: SET TO should be an integer
+AT_CHECK([$COMPILE -x -std=mf prog.cob], [0], [], [prog.cob:25: warning: SET TO should be an integer
```

Comment:
we want to run this and don't care about the warnings here - disable by whatever option is best matching and drop `-x` and - if possible - also the `-std`